ui/js/directives/treeherder/clonejobs.js

```diff
@@ -526,29 +566,29 @@ treeherder.directive('thCloneJobs', [
 
             //Empty the job column before populating it
             jobTdEl.empty();
-
-            var jgObj, jobGroup, i;
+            var jgObj, jobGroup, i, jobTdHtml = "";
             for (i=0; i<jobGroups.length; i++) {
```

Comment:
Since we're modifying so much here anyway, it would be nice to just use a `forEach` here, that would let us get rid of the `i` temporary.